packages/blade/src/components/ActionList/ActionListItem.tsx

```diff
@@ -105,12 +106,19 @@ const _ActionListSection = ({
   testID,
   _hideDivider,
 }: ActionListSectionProps): JSX.Element => {
+  const { theme } = useTheme();
+
   return (
     <BaseBox
       {...makeAccessible({
         role: getActionListSectionRole(),
         label: title,
       })}
+      style={{
+        paddingTop: theme.spacing[3],
+        marginTop: -theme.spacing[3],
+      }}
```

Comment:
Can use direct props from Box right?
```suggestion
      paddingTop="spacing.3"
      marginTop={`-${theme.spacing[3]}px`}
```

Reply:
Oh I think this is messing up with spacings of its upper element? The padding of element above section header gets cropped out.

<img width="356" alt="image" src="https://user-images.githubusercontent.com/30949385/232440226-456db2e1-6770-4807-9a95-2feaffb7db00.png">

<img width="579" alt="image" src="https://user-images.githubusercontent.com/30949385/232440330-8f490b7a-a91f-4189-b63e-b004e0720495.png">

Reply:
Will check.

Reply:
Fixed.